config/arkscan.php

```diff
@@ -55,7 +55,9 @@
             'blockReward'         => intval(env('ARKSCAN_NETWORK_BLOCK_REWARD', 2)),
             'base58Prefix'        => intval(env('ARKSCAN_NETWORK_BASE58_PREFIX', 30)),
 
-            'contract_addresses' => [],
+            'contract_addresses' => [
+                'consensus' => env('ARKSCAN_CONSENSUS_CONTRACT_ADDRESS', '0x535B3D7A252fa034Ed71F0C53ec0C6F784cB64E1'),
```

Comment:
will this data work with the `isContract` logic that expects an array of addresses instead?

Reply:
It should yeh but I'll add a test to the MemoryWallet to be sure